models/articles.models.js

```diff
@@ -40,4 +40,27 @@ function selectAllArticles() {
     });
 }
 
-module.exports = { selectArticlesId, selectAllArticles };
+function selectCommentsFromArticle(articleID) {
+  if (!/^[0-9]+$/.test(articleID)) {
+    return Promise.reject({ status: 400, msg: "invalid id" });
+  }
```

Comment:
I see what you did, however, it isn't needed. sql will throw an error if the article id is not a number and you can use the code to throw an error in your error handler